test/users.js

```diff
@@ -171,3 +171,16 @@ describe('Test for registering a new user', () => {
     res.body.message.should.equal('This email has been taken');
   });
 });
+
+
+describe('Test for listing all users', () => {
+  it('Should return status 200 on successful retreival Of users', async () => {
+    const response = await chai.request(app)
```

Comment:
`of users` not `Of users`

Reply:
Thanks for pointing this out, it will be resolved shortly.